app/models/project.rb

```diff
@@ -1,4 +1,6 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
```

Comment:
Nice work setting up the many to many relationship! Just be sure to test it in model specs!